lib/sidekiq/cronitor.rb

```diff
@@ -27,16 +27,19 @@ def call(worker, message, queue)
     end
 
     private
+
     def cronitor(worker)
       Cronitor::Monitor.new(job_key(worker))
     end
 
     def cronitor_disabled?(worker)
-      disabled = worker.class.sidekiq_options.fetch("cronitor_disabled", nil)
-      if disabled.nil?
-        options(worker).fetch(:disabled, false)
+      enabled = worker.class.sidekiq_options.fetch("cronitor_enabled", false)
```

Comment:
Is the `cronitor` hash legacy? Do we still need to support it here?

Reply:
iirc when version 3.0 was written it was originally added, but there was an issue with reading nested hashes out of sidekiq options (i don't really recall what/how it manifested), so it was dropped from the explicit `cronitor_xxx` pattern.